### ISSUE TITLE
Add support for plain inline buttons

### ISSUE THREAD

**Issue by marshmalien** [patternfly/patternfly-react#3750]
#### SUMMARY
Button Variations: https://www.patternfly.org/v4/documentation/react/components/button#variations

The button prop `isInline` currently doesn't play well with `variant="plain"`.  In the example below, the DataListAction contains a plain button, but it adds about 10px of whitespace to the row height. 

Changing the button variant to `link` and adding `isInline` removes the extra whitespace, but we don't want the "link" styles. Could you add support for a "plain" "inline" button?

##### Plain 
<img width="501" alt="Screen Shot 2020-02-13 at 4 23 12 PM" src="https://user-images.githubusercontent.com/15881645/74479547-56102a80-4e7d-11ea-9fa5-52e5f4bd0fec.png">

##### Link and Inline
<img width="447" alt="Screen Shot 2020-02-13 at 4 22 47 PM" src="https://user-images.githubusercontent.com/15881645/74479534-527ca380-4e7d-11ea-918b-b12c3037d222.png">

**Comment by tlabaj** [comment]
@mcoker Do you have any recommendations here?

**Comment by mcoker** [comment]
Sounds good to me. @mcarrano is there a reason we wouldn't want to support an inline variation of the plain button?

**Comment by mcarrano** [comment]
I don't think so @mcoker . So is this just a variant with different spacing then?

**Comment by mcoker** [comment]
@mcarrano yes. I'll move this issue to our backlog.
